_includes/footer.html

```diff
@@ -19,10 +19,10 @@
       </div>
       <!-- copyright info -->
       <div class="twelve columns site-footer-cc">
-        <p>&#169; 2016 Toronto Mesh. Our <a href="/code-of-conduct/">Code of Conduct</a>.<br />This website content is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International</a> license.</p>
+        <p> &copy;<script>new Date().getFullYear()>2010&&document.write(" "+new Date().getFullYear());</script>, Toronto Mesh. Our <a href="/code-of-conduct/">Code of Conduct</a>.<br />This website content is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International</a> license.</p>
```

Comment:
@garrying @dcwalk I think we prefer generating this into static text with jekyll than js? That can be done right?